service/history/workflowExecutionContext.go

```diff
@@ -379,6 +391,14 @@ func (c *workflowExecutionContextImpl) conflictResolveWorkflowExecution(
 		if err != nil {
 			return err
 		}
+		if conflictResolveMode == persistence.ConflictResolveWorkflowModeBypassCurrent {
```

Comment:
plz merge the 2 events reapply in one place and preferable a dedicated func

Reply:
done